README.md

```diff
@@ -24,15 +24,12 @@ Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/S
 
 # Prerequisites for Installation
 
- * LINUX (supported on Ubuntu 14+ and [CentOS 7](https://github.com/aces/Loris/blob/master/README.CentOS6.md))
  * Apache **2.4** or higher
  * MySQL 5.7
  * PHP <b>7.2</b> or higher
  * Package manager (for LINUX distributions)
- * Composer : should be run with --no-dev option
```

Comment:
composer is still required. But maybe the line above it about needing a package manager should be removed while we're at it?

Reply:
I removed this line because it is basically a duplicate from line 36.

I'll remove the line above.

Reply:
It would make more sense to me to keep it here and delete it below, because installing it is still a prerequisite.